lib/libprocstat/core.c

```diff
@@ -70,9 +70,6 @@ __FBSDID("$FreeBSD$");
 #ifndef EF_MIPS_MACH_CHERI128
 #define	EF_MIPS_MACH_CHERI128	0x00C10000
 #endif
-#ifndef	EF_MIPS_MACH_CHERI256
-#define EF_MIPS_MACH_CHERI256	0x00C20000
-#endif
 #ifndef EF_MIPS_MACH
 #define	EF_MIPS_MACH		0x00FF0000
 #endif
```

Comment:
Shouldn't all of these already be defined by sys/elf.h?

Reply:
Do gelf.h/libelf.h end up pulling that in or just define their own set of types and macros?

Reply:
gelf.h and libelf.h have their own macros, but I'd rather us patch those over in contrib and not need the #ifndef here, but that probably warrants a separate cleanup.